README.md

```diff
@@ -110,6 +110,69 @@ First, you'll want to have Node.js installed. You can do this by going to [nodej
 
 10. Go to your forked repository on GitHub and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
 
+## Keeping Branch Up to Date and Resolving Merge Conflicts
+
+Due to open source collaboration culture, it is common to encounter merge conflicts when contributing to an open source project. Merge conflicts usually occur when there are changes on the same line(s) in the same file(s) from two different branches.
```

Comment:
[suggestion] It's a good start, but it could be a bit more casual and clearer. Remember, most of the contributors are beginners, so we need to convey this information in an easy-to-read format. 
```suggestion
Merge conflicts are something you'll encounter a lot when contributing to an open source project. They usually occur when there are changes on the same line(s) in the same file(s) from two different branches.
```

Reply:
In that case, I rather use the word "common" than "a lot".
For beginners in open source, merge conflicts are something terrifying. So reading the word "a lot" can discourage them if they don't have smooth first experience.

How about: 

Merge conflicts are something you'll *commonly* encounter when contributing to an open source project. They usually occur when there are changes on the same line(s) in the same file(s) from two different branches.